prow/config/config.yaml

```diff
@@ -323,6 +323,9 @@ tide:
     pingcap/tidb: squash
 
   merge_commit_template:
+    ti-community-infra/tichi:
```

Comment:
Can you help to see if this supports using org names for all repository configurations? Not just tichi.

Reply:
Support

![image](https://user-images.githubusercontent.com/5086433/108943193-4a139700-7693-11eb-94f9-5fded14595df.png)

Reply:
Cool